sources/@roots/bud-tailwindcss-theme-json/src/tailwind/palette.ts

```diff
@@ -34,7 +34,7 @@ export interface transformEntry {
   (slug: Array<string>, color: string): WordPressColors[any]
 }
 export const transformEntry: transformEntry = (slug, color) => ({
-  color: color.toLowerCase(),
+  color: color.toLowerCase().replace(/\/ <alpha-value>/g, '/ 1'),
```

Comment:
```suggestion
  color: color.toLowerCase().replace(/<alpha-value>/g, `1`),
```

I think this regexp should be modified to support `rgba` syntax in addition to `rgb` and `hsl`. 

The quotes also need to be backticks or this will fail lint checks.